.github/workflows/CD.yml

```
@@ -0,0 +1,40 @@
+name: CD
+
+on:
+    push:
+      tags:
+        - '*'
```

I think this can be simplified to the following:
```suggestion
      tags:
```

---

I don't think that's an option. 

Actually, I am going to change this to:
```
tags:
    - '**'
```
to also include hierarchical tags (if we ever plan to use hierarchical tags anyway) such as `1.1/alpha`, etc.
https://stackoverflow.com/a/61892639/21024780

---

Fair enough!